mudslide/periodic_table.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+
+
+from __future__ import print_function
+
+import sys
+
+import math as m
+import numpy as np
+
+DEBUG = False
+
+# masses of most common isotopes to 5 decimal points, from
+# http://physics.nist.gov/cgi-bin/Compositions/stand_alone.pl
+masses = {
```

Comment:
Use all of the digits that the nist table provides. No reason to cut off at 5 decimal points.

Reply:
addressed